Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
 # Multiplayer Samples Co-op Changelog
 
+## [0.2.0] - 2021-05-18
+
+v0.2.0 is an Early Access release for Multiplayer Samples Co-op.
+
+It requires and supports Unity v2020.3 and later and Unity MLAPI v0.1.0. For additional information on MLAPI, see the changelog and release notes for the Unity MLAPI package.
```

Comment:
Thinking we should list the actual version tested with 2020.3.8 LTS - also reword the "and later" because we haven't tested against that, but it could work

Reply:
Updating versions.